pkg/controller/gittrack/gittrack_controller.go

```diff
@@ -542,7 +542,7 @@ func (r *ReconcileGitTrack) Reconcile(request reconcile.Request) (reconcile.Resu
 
 	// Update status with the number of objects discovered
 	sOpts.discovered = int64(len(objects))
-	// Get a list of the GitTrackObjects that currently exist, by name
+	// Get a list of the GitTracks that currently exist, by name
```

Comment:
This one is also correct. We list the existing `GitTrackObjects` that are owned by the current `GitTrack` and then we reconcile the difference between the existing `GitTrackObjects` and the ones we wish to create based off of the git repository state